.travis.yml

```diff
@@ -25,7 +25,7 @@ matrix:
           group: edge
         - php: 5.5
         - php: 5.6
-        - php: 7.0
+        - php: 7.0.8
```

Comment:
We should test against the latest 7.0 too, to benefit from PHP bug fixes

Reply:
7.0 === 7.0.7 so we cannot have that line right now

Reply:
but we could add another build with `7.0.23` (granted, it would not automatically bump to the latest release).

Btw, on Precise, we were getting an uptodate version because it was not preinstalled (and so the 7.0 alias was resolved when starting the build rather than when building the Travis image)